addons/web/static/src/js/fields/special_fields.js

```diff
@@ -100,6 +100,18 @@ var FieldTimezoneMismatch = FieldSelection.extend({
             this._renderDateTimeTimezone();
         }
     },
+    // @override
+    // this.$el can have other elements than select
+    // that shouldn't be touched
+    _renderEdit: function () {
```

Comment:
correctly format docstring

Reply:
I'm not a fan of the fix. Widgets with multiple root elements are a bad idea and error prone. However, fixing this correctly would require much more changes/refs, so i agree to do it that way, if you add a full explanation of why you're doing this (+ a big FIXME with what i said).